classes/Forms.js

```diff
@@ -230,6 +230,26 @@ module.exports = class Forms extends OneBlinkAPI {
     })
   }
 
+  async createForm(data /* : ?mixed */) /* : Promise<Form> */ {
+    const form = validateWithFormSchema(data)
+    const savedForm = await super.postRequest('/forms', form)
+    return savedForm
+  }
+
+  async updateForm(data /* : ?mixed */) /* : Promise<Form> */ {
+    const form = validateWithFormSchema(data)
+    const savedForm = await super.putRequest(`/forms/${form.id}`, form)
+    return savedForm
+  }
+
+  async deleteForm(formId /* : ?mixed */) /* : Promise<void> */ {
```

Comment:
How come this has to be mixed?

Reply:
it is a public function, anyone can pass what ever they want to it.

Reply:
ah yeh